### PR TITLE
[BOUNTY] makes tourist only available to new players and donators

### PR DESCRIPTION
**Pull request by Moltijoe**
https://forums.yogstation.net/threads/10-make-changes-to-the-tourist-role-ss13-server.22273/

# Why is this good for the game?
read the bounty

# Testing
image
image

:cl:  
tweak: Tourist can no longer have fisher as an alternative name
tweak: Assistant can now have fisher as an alternative name
tweak: Tourist is now only available to donators or players under 50 hours old
tweak: Tourist now gets quiet mode
/:cl:

### CODE REVIEW

**Review by ToasterBiome — APPROVED** [review]

**Review by ynot01 — APPROVED** [review]

**Comment by ynot01** [comment]
`Add a notification before joining that informs the player of the above critical information`

**Comment by ktlwjec0** [comment]
Does this fix #22460 , if not could you add a fix in with this PR, it probably just needs map spawn points for tourist or making assistant spawns count for them too. Just this PR makes it an even bigger confusion for new players otherwise.

**Comment by N3D6** [comment]
This is so pointless.

**Comment by Moltijoe** [comment]
> Does this fix #22460 , if not could you add a fix in with this PR, it probably just needs map spawn points for tourist or making assistant spawns count for them too. Just this PR makes it an even bigger confusion for new players otherwise.

Honestly, probably not going to waste any more time on this pr

**Comment by Moltijoe** [comment]
> This is so pointless.

Care to explain why?

**Comment by N3D6** [comment]
> > This is so pointless.
> 
> Care to explain why?

Yes. There isn’t a point to this. we don’t have a problem with tourists running around greyshitting, this is literally just ruining a role for shits and giggles because LITERALLY everyone who wants a learning role goes assistant anyway

**Comment by Moltijoe** [comment]
> Yes. There isn’t a point to this. we don’t have a problem with tourists running around greyshitting, this is literally just ruining a role for shits and giggles because LITERALLY everyone who wants a learning role goes assistant anyway

In my eyes this is more about providing quiet mode to new players without needing to be a donator

**Comment by Cartlord** [comment]
do you plan on actually adding something that indicates to new players that tourist is the new player role before they click the role

**Comment by Theonecarpman** [comment]
This is honestly useless, There's tourist mains who have well above 50 hours and actual enjoy it, this honestly seems to just be restricting the role itself to mainly donators in my eyes.

**Comment by MegaEmpirical** [comment]
Glorious. I'd suggest adding to the notification about how it's only for under 50 hour players and donors.

**Comment by Junkyshell** [comment]
good pr

**Comment by StupidTums** [comment]
I don't agree for this role to be available to donators. The uniqueness should come from the fact that a tourist will always be a new player, and this is giving them a temporary free option of quiet mode.

**Comment by athena413** [comment]
if you really want tourist to be a new player role you should make every single other role have like a 5 minute "time as crew" playtime requirement or some shit so that new players have to play as tourist for their first round

File: code/modules/client/preferences.dm [hunk 1]
```diff
@@ -475,14 +475,17 @@ INITIALIZE_IMMEDIATE(/atom/movable/screen/character_preview_view)
 	if (!job)
 		return FALSE
 
+	if(job.title == "Tourist" && level != null) //yogs change: tourist is newbie players only
+		if(tgui_alert(parent, "Tourist is a job for exploring the station at your own pace.\n You can never roll antagonist, or be the target for one.", "Enable Tourist mode?", list("Yes", "No"), 30 SECONDS) != "Yes")
```
Comment (alexkar598) [inline code/modules/client/preferences.dm:479]:
```suggestion
		if(tgui_alert(parent, "Tourist is a job for exploring the station at your own pace.\n This mode prevents you from being chosen as a target for traitors but you will be ineligible for randomly selected antagonist roles.", "Enable Tourist mode?", list("Yes", "No"), 30 SECONDS) != "Yes")
```

**Review by alexkar598 — COMMENTED** [review]
Since this is aimed at new players, terminology such as "roll" and "antagonist" may be unclear. The not being a target is also more relevant for new players than the not being an antag

**Comment by Moltijoe** [comment]
quickfire votes says:
no

**Pull request closed without merging**